herbert_core/utilities/read_write/serialize_variable.m

```diff
@@ -20,7 +20,7 @@
 % data access routines to be used to read the file. Normally just
 % use the built-in Matlab load and save routines.
 
-class_name = class(var);
+type = class(var);
```

Comment:
does this need to be evaluated here, or could it move into the `isstruct(var)` block

Reply:
Its result is used recursively in 3 out of 4 if-branches. One can, of course, transfer it into each out of 3 necessary, but performance benifits are insignificant and mainenance costs increases.
I believe it was `strcpmi(type,'cell')` in the first line of the branch but Matlab just feels suspishious about it.